usaspending_api/awards/v2/filters/sub_award.py

```diff
@@ -90,8 +90,9 @@ def keyword_parse(keyword):
             logger.info('Found {} transactions based on keyword: {}'.format(len(transaction_ids), keyword))
             transaction_ids = [str(transaction_id) for transaction_id in transaction_ids]
             queryset = queryset.filter(latest_transaction_id__isnull=False)
-            queryset &= queryset.extra(
-                where=['"latest_transaction_id" = ANY(\'{{{}}}\'::int[])'.format(','.join(transaction_ids))])
+
+            sql_fragment = '"subaward_view"."latest_transaction_id" = ANY(\'{{{}}}\'::int[])'
+            queryset &= queryset.extra(where=[sql_fragment.format(','.join(transaction_ids))])
```

Comment:
I see no protection against SQL injection here, and Django's documentation on .extra clearly states it DOESN'T protect against that. https://docs.djangoproject.com/en/2.1/ref/models/querysets/

Reply:
Good eye on using .extra(). The list of `transaction_ids` are from the ES index, not user-provided. So risk of SQL injection is very low